odoo/addons/base/models/res_partner.py

```diff
@@ -480,7 +479,7 @@ def _commercial_fields(self):
         partners that aren't `commercial entities` themselves, and will be
         delegated to the parent `commercial entity`. The list is meant to be
         extended by inheriting classes. """
-        return ['vat', 'credit_limit']
+        return ['vat']
```

Comment:
reading this function, I'm thinking of the following: in its current implementation, I have the impression that the credit_limit applies on a partner and isn't depending on their entity.

For example, let's say there is JOL from Odoo and BIB from Odoo too.
BIB has already reached the credit limit.
Does the credit limit from BIB applies to JOL too as they are both from Odoo or does it apply only to BIB and JOL can have his own credit limit?

Reply:
The field is set on the partner (BIB or JOL or Odoo) in your example.
So you choose where you want it to be set, if you want every one from Odoo to have the same limit, you set it on Odoo.
But if you know that JOL is a pretty weird dude and you don't trust him so much, you can set a limit just for him :wink:

Reply:
> The field is set on the partner (BIB or JOL or Odoo) in your example.

It depends how you check it. There is also a field `commercial_partner_id` that could be used to get it for Odoo instead of JOL or BIB
https://github.com/odoo/odoo/blob/5b72f017662525362ff1e1bc8173a7ab395dbcf9/odoo/addons/base/models/res_partner.py#L227-L230

The question is still open, but the implementation is exactly the same, except that you need to look on the partner or on its commercial entity (and that the field should then be visible or not on the contacts)

Reply:
I've adapted the behavior to use the `commercial_partner_id` instead.
Thanks for the heads up :slightly_smiling_face: